exchanges/gateio/gateio_websocket.go

```diff
@@ -43,11 +45,35 @@ func (g *Gateio) WsConnect() error {
 		return err
 	}
 
+	if g.AuthenticatedAPISupport {
+		err = g.wsServerSignIn()
+		if err != nil {
+			log.Errorf("Authentication failed ")
```

Comment:
Would be better to have a more descriptive error message, especially if the context has with multiple exchanges loaded.

`log.Errorf("%s websocket authentication failed. Err: %v", g.Name, err)`